services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/nodeports.py

```diff
@@ -243,14 +244,28 @@ def _is_zip_file(file_path: Path) -> bool:
 _shutil_move = aiofiles.os.wrap(shutil.move)
 
 
+async def _move_file_to_input_port(
+    final_path: Path, downloaded_file: Path, dest_folder: PrunableFolder
+) -> None:
```

Comment:
this function named `_move_file_to_input_port` takes arguments that do not have any port in their name. even the function does not check anything port related...
actually I don't even really understand what `dest_folder` is? was that the original folder where the downloaded_file is living?

so this funciton
- moves downloaded_file into final_path
- then prunes dest_folder exluding final_path
maybe a bit of refactoring is in order? at least with naming the arguments?

Reply:
Let me fix this